cilium/cmd/bpf_auth_flush.go

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cilium/cilium/pkg/command"
+	"github.com/cilium/cilium/pkg/common"
+	"github.com/cilium/cilium/pkg/maps/authmap"
+)
+
+var bpfAuthFlushCmd = &cobra.Command{
+	Use:     "flush",
+	Short:   "Deletes all entries for authenticated connections between identities",
+	Long:    "Deletes all entries for authenticated connections between identities",
+	Aliases: []string{"ls"},
+	Run: func(cmd *cobra.Command, args []string) {
+		common.RequireRootPrivilege("cilium bpf auth flush")
+
+		authMap, err := authmap.LoadAuthMap()
+		if err != nil {
+			if errors.Is(err, fs.ErrNotExist) {
+				fmt.Fprintln(os.Stderr, "Cannot find auth bpf map")
+				return
+			}
+
+			Fatalf("Cannot load auth bpf map: %s", err)
+		}
+
+		entries := 0
+		deleteEntry := func(key *authmap.AuthKey, _ *authmap.AuthInfo) {
+			if key == nil {
+				return
+			}
+			authMap.Delete(*key)
```

Comment:
probably checking returned err here, and print out if not nil. Also, only do entries++ if there is no error.

Reply:
good point!